site/pages/ajax/sitemenu-v5-en.hbs

```diff
@@ -12,8 +12,8 @@ language: "en"
 		<li role="separator"></li>
 		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/jobs/opportunities.html">Find a job</a></li>
 		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/jobs/training.html">Training</a></li>
-		<li role="presentation"><a role="menuitem" tabindex="-1" href="http://www.canada.ca/business-management">Hire and manage employees</a></li>
-		<li role="presentation"><a role="menuitem" tabindex="-1" href="http://www.canada.ca/start-business">Starting a business</a></li>
+		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/business-management">Hire and manage employees</a></li>
+		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/start-business">Starting a business</a></li>
```

Comment:
We will need to do apply the same update on our WMS hosted and CDN version of the menu.

/cc @thangle2013 @therealolivergross @GormFrank

Reply:
This has been done in a recent JIRA ticket